### PR TITLE
doc: add guides zh-docs

### PR DESCRIPTION
**Pull request by tianweidut**
## Modules
- [x] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [x] add necessary doc

### CODE REVIEW

File: docs/docs/guides/install/standalone.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ sudo apt-get update
 sudo apt-get install -y python3.7 python3.8 python3.9 python3-pip python3-venv python3.8-venv python3.7-venv python3.9-venv
 ```
 
-Starwhale works on macOS X. If you run into issues with the default system Python3 on macOS, try installing Python3 through the homebrew:
+Starwhale works on MacOS. If you run into issues with the default system Python3 on MacOS, try installing Python3 through the homebrew:
```
Comment (jialeicui) [inline docs/docs/guides/install/standalone.md:31]: `macOS` is the correct spelling
https://en.wikipedia.org/wiki/MacOS

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+```console
++-- ${storage.root}
+|  +-- .objectstore     --> 存储数据集chunk文件的简单存储，使用blake2b hash算法
+|  |   +-- blake2b      --> hash算法名称
+|  |   |  +-- 00        --> hash2位前缀
+|  |   |  |  +-- 0019ad58...   --> 完成的object文件，文件名就是文件内容的hash值
+|  |   |  +-- 05
+|  +-- .datastore       --> 基于pyarrow的列式存储
+|  |   +-- project
+|  |   |   +-- self     --> 按照project 名称进行分类存储
```
Comment (jialeicui) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md:76]: 移除project 后的空格?

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## 1. config.yaml 说明
+
+典型的config.yaml文件内容如下，表示当前默认默认Instance为local，管理4个Instance，其中cloud-cn/cloud-k8s/pre-k8s三个为Cloud Instance，local为Standalone Instance。Standalone本地存储的根目录为 `/home/liutianwei/.starwhale` ：
+
+```yaml
+current_instance: local
+instances:
+  cloud-cn:
+    sw_token: Bearer TOKEN
```
Comment (goldenxinxing) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md:17]: sw_token: Bearer TOKEN -> sw_token: Bearer {TOKEN}

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+## 2. Standalone Instance的文件存储结构
+
+${storage.root} 目录中存储了Starwhale Standalone Instance所有的用户数据，包括Project、Runtime、Model、Dataset、Evaluation等用户直接感知的数据，也包括ObjectStore、DataStore等Starwhale后台实现的存储。具体说明如下：
+
+```console
++-- ${storage.root}
+|  +-- .objectstore     --> 存储数据集chunk文件的简单存储，使用blake2b hash算法
+|  |   +-- blake2b      --> hash算法名称
+|  |   |  +-- 00        --> hash2位前缀
+|  |   |  |  +-- 0019ad58...   --> 完成的object文件，文件名就是文件内容的hash值
```
Comment (goldenxinxing) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/config/standalone_config.md:72]: 是要表达 完成->完整 还是已完成呢

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/install/helm-charts.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+| `externalMySQL.password` | 外部的MySQL 密码 | |
+| `externalMySQL.database` | 外部的MySQL 的database 名称 | `starwhale` |
+| `minio.enabled` | 部署一个单例的Minio服务，需要为Minio提供一个PV持久化数据| `true`|
+| `minio.persistence.storageClass` | minio pvc storageClass | `local-storage-minio` |
+| `externalS3OSS.host`| 当 minio.enabled 设置为 false, 使用外部的基于S3协议的对象存储的host | `localhost` |
+| `externalS3OSS.port`| 外部对象存储的port | `9000` |
+| `externalS3OSS.accessKey`| 外部对象存储的access key |  |
+| `externalS3OSS.secretKey`| 外部对象存储的secret key | |
+| `externalS3OSS.region`| 外部对象存储的region | `local` |
+| `externalS3OSS.type`| 外部对象存储的类型，目前支持s3和aliyun两种 | `s3` |
```
[the remark this answers is not in the document]
Comment (jialeicui) [reply docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/install/helm-charts.md:71]: And the default is minio
https://github.com/star-whale/starwhale/blob/d351d90ec71347c5e0a87af88046243eff7354d1/server/storage-access-layer/src/main/java/ai/starwhale/mlops/storage/configuration/StorageAccessConfig.java#L46-L50

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/dataset.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+### 3.1 YAML 字段描述
+
+|字段|描述|是否必要|类型|默认值|
+|---|---|-------|---|-----|
+|name|Starwhale Dataset的名字|是|String||
+|handler|继承 `starwhale.SWDSBinBuildExecutor`, `starwhale.UserRawBuildExecutor` 或 `starwhale.BuildExecutor` 类的可import地址，格式为 {module 路径}:{类名} |是|String||
+|desc|数据集描述信息|否|String|""|
+|version|dataset.yaml格式版本，目前仅支持填写 1.0|否|String|1.0|
+|pkg_data|swds中包含的文件或目录，支持wildcard方式描述。默认会包含 `.py/.sh/.yaml` 文件|否|List[String]||
+|exclude_pkg_data|swds中排除的文件或目录，支持wildcard方式描述。默认会包含 `.py/.sh/.yaml` 文件|否|List[String]||
```
[the remark this answers is not in the document]
Comment (tianweidut) [reply docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/dataset.md]: no

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**